src/mqt/yaqs/core/data_structures/noise_model.py

```
@@ -166,6 +167,45 @@ def __init__(
+        """
+        new_processes: list[dict[str, Any]] = []
+        for proc in self.processes:
+            new_proc = proc.copy()
+            strength_val = proc["strength"]
+
+            if isinstance(strength_val, dict):
+                dist_type = strength_val.get("distribution", "normal")
+                if dist_type == "normal":
+                    mean = strength_val.get("mean", 0.0)
+                    std = strength_val.get("std", 0.0)
+                    sampled_val = np.random.normal(loc=mean, scale=std)  # noqa: NPY002
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `NPY002`)

Remove unused `noqa` directive

(RUF100)